packages/panda-plugins/src/strict-tokens-scope.ts

```diff
@@ -80,14 +81,23 @@ export const transformPropTypes = (
   if (!strictTokenProps.length) return args.artifacts
 
   if (logger) {
-    logger.debug('plugin:restrict-strict-tokens', `🐼  Strict token props: ${strictTokenProps.join(', ')}`)
+    logger.debug('plugin:restrict-strict-tokens', `🐼  Exclude token props: ${strictTokenProps.join(', ')}`)
   }
 
   // const regex = new RegExp(`(${strictTokenProps.join('|')})\?: ConditionalValue<WithEscapeHatch<(.+)>>`, 'g')
   const regex = /(\w+)\?: ConditionalValue<WithEscapeHatch<(.+)>>/g
 
   const transformed =
     content.code.replace(regex, (match, prop, value) => {
+      if (shouldExclude) {
```

Comment:
in my mind, this feature could be done simply by re-using the current logic in `transformPropTypes` and filling the `categories` with every possible `TokenCategory` (when `excludeCategories` is not empty) then filtering the array using `excludeCategories`

something like:
```ts
const allCategories = ["zIndex", "opacity", "colors", "fonts", "fontSizes", "fontWeights", "lineHeights", "letterSpacings", "sizes", "shadows", "spacing", "radii", "borders", "durations", "easings", "animations", "blurs", "gradients", "assets", "borderWidths", "aspectRatios", "containerNames"]

// excludeCategories: ['colors', spacing'] -> allCategories.filter((cat) => !excludeCategories.include(cat))
```